### PR TITLE
patch: Add etcherPro documentation

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Moving etcherPro docs from balena-site

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

**Comment by builder555** [comment]
@vipulgupta2048 can you fix the issue with Docusaurus? then I can approve the PR

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- Plug one or more drives you would like to backup into the slot(s) with the blue-colored blinking LED
+- The drives will be autoselected and the LED will stop blinking
+- Select **OK** to move on to the next step
+- Plug the backup drive into the slot with the white-colored blinking LED. The plugged target will be automatically selected and the LED will stop blinking
+- Select **Flash**, to begin the flashing process
+- The LED of the target slot will first blink purple for flashing, and then green for validating
+- Once the flashing is complete, the LED will turn green for a successful flash, and red for a failed one
+- You may safely unplug the drive when flashing is complete
+
+### Format drive
```
Comment (builder555) [inline docs/getting-started.md:94]: ATM there is no format option on EtcherPro, can you remove this section please?

Comment (vipulgupta2048) [reply docs/getting-started.md:94]: Will do

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+- From the Etcher menu, select **Clone drive**
+- Plug a drive you would like to clone into the slot with the blue-colored blinking LED
+- The drive will be selected automatically and the LED will stop blinking
+- Plug at least one drive into an available slot. The plugged targets will be selected automatically, and the LEDs will turn white
+- Select **Flash**, to begin the flashing process
+- The LED of each slot will first blink purple for flashing and then green for validating
+- Once the flashing is complete, the LEDs will turn green for successfully flashed drives, and red for the failed ones
+- You may safely unplug the drives when flashing is complete
+
+### Backup drive
```
Comment (builder555) [inline docs/getting-started.md:80]: this section also needs to be removed, sorry, i missed it the first time around. @vipulgupta2048

Comment (vipulgupta2048) [reply docs/getting-started.md:80]: Cool

**Comment by Bucknalla** [comment]
Hey @vipulgupta2048 @builder555 shouldnt this PR live in this repo -> https://github.com/balena-io-hardware/etcherPro-info-doc? CC @hraftery

**Comment by Bucknalla** [comment]
@vipulgupta2048 I think according to Heath's new naming strategy is that it should go into the `etcherPro-info-doc` repo but I'll wait for him to weigh in @hraftery

**Comment by vipulgupta2048** [comment]
Thanks for the clarification.

**Comment by hraftery** [comment]
@vipulgupta2048 the strategy (hot off the press, documentation on the way!) is that repos are single-purpose and named accordingly. To give you a picture, here are the current EP repos:

- etcherPro-assemblyGuide-doc
- etcherPro-cardReader-pcb
- etcherPro-emmcMicroSd-pcb
- etcherPro-emmcSd-pcb
- etcherPro-enclosure-3d
- etcherPro-flashStick-pcb
- etcherPro-fleet-sw
- etcherPro-flexAntennaDualBand-doc
- etcherPro-info-doc
- etcherPro-mainboard-pcb
- etcherPro-mainboardTestBed-3d
- etcherPro-mainboardTestBed-pcb
- etcherPro-marketingMaterial-doc
- etcherPro-packaging-2d
- etcherPro-usbSdSlotPc-pcb
- etcherPro-userManual-doc

It looks to me like you're adding a user manual? I can't quite tell at a glance, which is why explicit naming is a great way to organise work. If so, `etcherPro-userManual-doc` is where you want to be. If it's actually a getting started guide that's different from the user manual, then a new repo like `etcherPro-gettingStarted-doc` is fine. Adding an entry to the readme at `etcherPro-info-doc` (which is sort of like an internal front door to the product) would be welcome, particularly if it's going to be published as the public facing landing page.

**Comment by vipulgupta2048** [comment]
Context: 
Atm, I am migrating docs from the balena-site marketing website to respective product repos. The etcherPro docs need to be migrated to their product repo (where the code lives): https://www.balena.io/etcher/pro/docs/getting-started

While I can see this landscape of a single product repo is not followed or can't be followed in balena-io-hardware org. I would rather the product owner decide where these docs ^ from the website should go to make the correct decision. 

TLDR: I don't know where this should go 😂 
Personal opinion from what Heath said: userManual seems right. But, since these will be public-facing then  `etcherPro-info-doc` seems like the best

@Bucknalla @hraftery @builder555 @(the product owner of EP?)

**Comment by builder555** [comment]
I guess that's where our confusion starts, currently the etcherPro manual *is* the "getting started" document. But ideally, I see the "getting started guide" being the first 1-2 pages of the full manual, so separating these into two repos is unnecessary. Having said that, I tend to agree with @hraftery, this document should go to the `etcherPro-userManual-doc` as long as that's where [this](https://www.balena.io/etcher/pro/docs/getting-started/) page will derive its content @vipulgupta2048

**Comment by vipulgupta2048** [comment]
That is correct, the documentation site being generated by this PR will be redirected here only: https://www.balena.io/etcher/pro/docs/getting-started/
